configs/cat_87_bmi_config_cfe_pass.txt

```diff
@@ -11,9 +11,15 @@ soil_params.expon_secondary=1.0[]
 max_gw_storage=0.25[m]
 Cgw=1.8e-05[m h-1]
 expon=6.0[]
+
+# gw_storage is the ground water as a decimal fraction of the maximum groundwater storage (max_gw_storage) for the initial timestep
```

Comment:
These comments are useful, but I think we'd benefit from consistency. I.e., we should comment all params in all configs, or we should save the comments/descriptions for a readme file.